src/w3c/templates/headers.js

```diff
@@ -7,7 +7,12 @@ export default conf => {
   const html = hyperHTML;
   return html`<div class='head'>
   ${conf.logos.map(showLogo)}
-  <h1 class='title p-name' id='title' property='${conf.doRDFa ? "dcterms:title" : null}'>${conf.title}</h1>
+  <script>
+    if (document.querySelectorAll("h1") ?
+      document.write("") :
```

Comment:
So, here we want to get just the first, so `const h1 = querySelector(“body>h1”)`.

Reply:
We want to avoid document.write (we are actually trying to remove it from the web entirely). It’s not allowed in many security contexts and has a huge performance penalty. 

Instead, use either the found h1, or just use the current code.

Reply:
Will provide you with some code suggestions once I’m back in front of a computer.

Reply:
> Will provide you with some code suggestions once I’m back in front of a computer

Fine I'll wait. 👍 Also, please check  Slack whenever you are free